### DISCUSSION TITLE
Please add an option to convert it into a month/year picker

### DISCUSSION THREAD

**Discussion by christeena-mary** [quasarframework/quasar#16337] [Ideas / Proposals]
I've added a lot of logic to turn it into a month/year picker, but there are still cases where the calendar view opens, and we can't control that. There are many date-pickers available that can be easily converted into a month-picker with a simple boolean flag. Unfortunately, this requirement came up towards the end of our project after we had already done everything else using Quasar, which has been quite challenging.

**Comment by metalsadman** [comment]
https://github.com/quasarframework/quasar/discussions/13101#discussioncomment-7016192
